client/config/environment.js

```diff
@@ -150,3 +151,10 @@ function getFeatureFlagSelfService() {
 function getFeatureFlagPhishingWarning() {
   return process.env.FEATURE_FLAG_PHISHING_WARNING === 'ON';
 }
+
+function getFeatureFlagExcludeFromSearchResults() {
+  if (process.env.FEATURE_FLAG_EXCLUDE_FROM_SEARCH_RESULTS === 'ON') {
+    return true;
+  }
+  return false;
```

Comment:
It doesn't functionally matter. But doesn't this just feel nicer? :relieved: 

```suggestion
  return process.env.FEATURE_FLAG_EXCLUDE_FROM_SEARCH_RESULTS === 'ON'
```